hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -274,7 +278,12 @@
       name: chip_usb_vbus
       desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
 
-            TBD.
+            - On the top of `chip_usb_fs_tx_rx`, add below at the end of the sequence.
```

Comment:
nit: on ~~the~~ top
or maybe slightly reformulated: "This test extends sequences from the `chip_usb_fs_tx_rx` test"